.github/workflows/build_and_deploy.yml

```
@@ -124,15 +129,16 @@ jobs:
           sudo apt update
           sudo apt install gcc-aarch64-linux-gnu g++-aarch64-linux-gnu
       - uses: nttld/setup-ndk@v1
-        if: matrix.target == 'aarch64-linux-android'
+        if: endsWith(matrix.target, '-linux-android')
         with:
           ndk-version: r25b
       - name: Set path for android
-        if: matrix.target == 'aarch64-linux-android'
+        if: endsWith(matrix.target, '-linux-android')
         shell: bash
         run: |
           echo "$ANDROID_NDK/toolchains/llvm/prebuilt/linux-x86_64/bin" >> "$GITHUB_PATH"
-          echo "AR_aarch64_linux_android=llvm-ar" >> "$GITHUB_ENV"
+          TRIPLE="${{ matrix.target }}"
+          echo "AR_${TRIPLE//-/-}=llvm-ar" >> "$GITHUB_ENV"
```

眺めていてこの部分が気になった（多分誤植？）のですが、CI が通っているのでちょっと不思議に思っています……（私の勘違いで、元の方が正しいということでしたらすみません）

```suggestion
          echo "AR_${TRIPLE//-/_}=llvm-ar" >> "$GITHUB_ENV"
```

---

あ、本当だ... なんで通るんでしょうか?

---

.cargo/config.tomlに書いているからだと思います。


…もしかしてこのActionなくても動く？